### PR TITLE
changed button to link and added title attribute

### PR DESCRIPTION
**Pull request by JoshHill15**
## Description

Changed button to link, added title attribute, and removed function that the button used to accomplish the same functionality.

## Testing

changed the unit test to check for title attribute with the correct value.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/containers/AdminHome.tsx [hunk 1]
```diff
@@ -154,10 +147,19 @@ function AdminHome() {
...
           </p>
-          <Button type="button" variant="outline" onClick={onViewPublicWebsite}>
-            {t("ViewPublishedSite")}{" "}
-            <FontAwesomeIcon size="sm" icon={faExternalLinkAlt} />
-          </Button>
+          <a
+            href="/"
+            target="_blank"
+            title="(opens in a new tab)"
+            className="link"
```
Comment (mabreuortega) [inline frontend/src/containers/AdminHome.tsx]: Why "link" and not "usa-link"?

**Review by mabreuortega — COMMENTED** [review]
Questions asked

File: frontend/src/containers/AdminHome.tsx (hunk 1 above)
Comment (JoshHill15) [reply frontend/src/containers/AdminHome.tsx]: I checked "Link.tsx" to see what class it was using and went off of that, I can change to "usa-link" if that's the preferred class for links.

Comment (mabreuortega) [reply frontend/src/containers/AdminHome.tsx]: yes, pls. .link was created for table action and the main different is they keep color constant for all the states (hover, active, visited)

**Review by miguelpdiaz8 — APPROVED** [review]

**Pull request merged**